http/jsonapi/runtime/parameters.go

```diff
@@ -90,6 +91,33 @@ func ScanParameters(w http.ResponseWriter, r *http.Request, parameters ...*ScanP
 			panic(fmt.Errorf("impossible scanning location: %d", param.Location))
 		}
 
+		// handle string and dates types
```

Comment:
Documentation of the surrounding function `ScanParameters` needs to be updated, because it says

> The used scanning function is fmt.Sscan

which is not true anymore. But it is an implementation detail and thus should not be part of the documentation, I guess.

Reply:
Updated the docs.